.gitignore

```diff
@@ -196,3 +196,5 @@ ModuleCache.noindex/
 Build/Intermediates.noindex/
 info.plist
 build-intermediate
+/tests/Avalonia.RenderTests/obj-Direct2D1
```

Comment:
This might be too specific, i wonder if both obj-Direct2D1 and obj-Skia folders exists in other projects as well?

Reply:
@jmacato You are right